File: MusicPlayer.DAL/Entities/Track.cs
namespace MusicPlayer.DAL.Entities
{
    public class Track : BaseEntity<int>
    {
        public string Name { get; set; }
        public string Summary { get; set; }
        public string Text { get; set; }
        public int AuthorId { get; set; }
        public string CoverUrl { get; set; }
        public string MusicUrl { get; set; }

        // Refrence to author
        public Author Author { get; set; }
    }
}

File: clientapp/src/models/Track.ts
/* eslint-disable @typescript-eslint/no-explicit-any */
export default class Track {
  id: number;
  name: string;
  summary: string;
  text: string;
  coverUrl: string;
  upload: any;
  musicUrl: string;
  authorId: number;
  howl: any | null;

  constructor() {
    this.id = 0;
    this.name = "";
    this.summary = "";
    this.text = "";
    this.coverUrl = "";
    this.upload = {};
    this.musicUrl = "";
    this.authorId = 0;
    this.howl = null;
  }
}
